### PR TITLE
add test of the gather_financial_statement_company_compare

### PR DESCRIPTION
**Pull request by sirine-chahma**
create a test file for this function and created the corresponding tests

### CODE REVIEW

File: tests/test_gather_financial_statement_company_compare.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    statement = 'income_statement'
+    year = '2014'
+    period = 'Q1'
+    
+    #Check that when the api_key is not a string, it returns an error
+    with raises(TypeError):
+        gather_financial_statement_company_compare(['la'], ticker, statement, year, period)
+    
+    #Check that when the statement is not a string, it returns an error
+    with raises(TypeError):
```
Comment (slbsolomon) [inline tests/test_gather_financial_statement_company_compare.py:25]: Can we make this specific so it raises an error when none of the valid input types #42 ?

**Review by slbsolomon — APPROVED** [review]
Comprehensive tests are included, which includes edge cases and expected values

File: tests/test_gather_financial_statement_company_compare.py (hunk 1 above)
Comment (slbsolomon) [reply tests/test_gather_financial_statement_company_compare.py:25]: Sorry for not being more specific - I had been referring to the statement type: income_statement, balance_sheet_statement, cash_flow_statement

I don't want to accept the changes listed because in the non-Free access versions, you can use any ticker, and this would prevent someone from doing that.

**Review by slbsolomon — APPROVED** [review]
all changes are as discussed
